src/Dobble/DobbleCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Dobble;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+
+
+class DobbleCommand extends Command {
+	protected function configure () {
+		$this->setName('dobble:run');
+		$this->setDescription('Command-line card generator for dobble-like game');
+
+		$this->addArgument(
+			'element_per_card',
+			InputArgument::REQUIRED,
+			'Number of element per card'
+		);
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$nb_element = $input->getArgument('element_per_card');
+
+		if($nb_element <= 0) {
+			$output->writeln('<error>Number of element must be a postive integer</error>');
+			return 0;
+		}
+
+		$output->writeln('Element per card: <info>' . (string)$nb_element . '</info>');
+	}
+}
```

Comment:
You must configure your text editor to add a newline at the end of the file - that's a convention.